src/artemis/parameters/internal_parameters.py

```diff
@@ -74,16 +75,56 @@ class InternalParameters:
     experiment_params: registry.EXPERIMENT_TYPES
 
     def __init__(self, external_params: RawParameters = RawParameters()):
-        self.artemis_params = ArtemisParameters(
-            **external_params.artemis_params.to_dict()
-        )
-        self.experiment_params = registry.EXPERIMENT_TYPE_DICT[
-            ArtemisParameters.experiment_type
-        ](**external_params.experiment_params.to_dict())
-        self.artemis_params.detector_params.num_images = (
-            self.experiment_params.get_num_images()
+        ext_expt_param_dict = external_params.experiment_params.to_dict()
+        ext_art_param_dict = external_params.artemis_params.to_dict()
+
+        omeg_inc = ext_expt_param_dict.get("omega_increment")
+        if omeg_inc is not None:
+            ext_art_param_dict["detector_params"]["omega_increment"] = omeg_inc
+            del ext_expt_param_dict["omega_increment"]
+        else:
+            ext_art_param_dict["detector_params"][
+                "omega_increment"
+            ] = ext_expt_param_dict["rotation_increment"]
```

Comment:
to remove it as a required experiment param? sure. I think it's probably good that it is included in the nexus file, better to have it be affirmatively 0 rather than not specified

Reply:
Yes, remove as a required param